client/main.js

```diff
@@ -32,10 +33,10 @@ window.addEventListener( 'beforeunload', ( event ) => {
 const rootEl = document.getElementById( 'wc-connect-admin-container' );
 
 let render = () => {
-	const WCCShippingSettings = require( 'views/shipping' );
+	const RootView = 'shipping-label' === rootView ? require( 'views/shipping-label' ) : require( 'views/shipping' );
```

Comment:
Better, but what about conditional rendering instead of conditional instantiation? Its more familiar a pattern.